awx/main/models/unified_jobs.py

```diff
@@ -882,6 +877,22 @@ def create_config_from_prompts(self, kwargs):
             config.credentials.add(*job_creds)
         return config
 
+    @property
+    def result_stdout_text(self):
+        try:
+            return self.deprecated_result_stdout.result_stdout_text
+        except ObjectDoesNotExist:
+            return ''
+
+    @result_stdout_text.setter
+    def result_stdout_text(self, value):
```

Comment:
This setter will need to continue to exist for the time being; my eventual goal once https://github.com/ansible/awx/issues/200 is fully implemented is for this method to totally go away (because no _new_ job runs should be storing stdout in this table).

Reply:
Might be good to mark the method for future deletion with a TODO?